pytorch_toolkit/light-weight-face-anti-spoofing/README.md

```diff
@@ -0,0 +1,124 @@
+* **dataset** - this is an indicator which dataset you will be using during training. Available options are 'celeba-spoof', 'LCC_FASD', 'Casia', 'multi_dataset', 'external'
+* **multi_task_learning** - specify whether or not to train with multitasking loss. **It is available for the CelebA-Spoof dataset only!**
+* **evaluation** - it is the flag to perform the assessment at the end of training and write metrics to a file
+* **test_dataset** - this is an indicator on which dataset you want to test. Options are the same as for dataset parameter
+* **external** - parameters for constructing external dataset reader. See Data Preparation section.
+* **img_norm_cfg** - parameters for data normalization
+* **scheduler** - scheduler for dropping learning rate
+* **data.sampler** - if it is true, then will be generated weights for `WeightedRandomSampler` object to uniform distribution of two classes
+* **resize** - resize of the image
+* **checkpoint** - the name of the checkpoint to save and the path to the experiment folder where checkpoint, tensorboard logs and eval metrics will be kept
+* **loss** - there are available two possible losses: `amsoftmax` with `cos`, `arcos`, `cross_enropy` margins and `soft_triple` with different number of inner classes. For more details about this soft triple loss see in [paper](https://arxiv.org/pdf/1909.05235.pdf)
+* **loss.amsoftmax.ratio**a  - there is availability to use different m for different classes. The ratio is the weights on which provided `m` will be divided for a specific class. For example ratio = [1,2] means that m for the first class will equal to m, but for the second will equal to m/2
+* **loss.amsoftmax.gamma** - if this constant differs from 0 then the focal loss will be switched on with the corresponding gamma
+* **For soft triple loss**: `Cn` - number of classes, `K` - number of proxies for each class, `tau` - parameter for regularisation number of proxies
+* **model** - there are parameters concern model. `pretrained` means that you want to train with the imagenet weights (you can download weights from [google drive](https://drive.google.com/drive/u/0/folders/1A6wa3AlrdjyNPkXT81knIzXxR7SAYm1q) and specify the path to it in the `imagenet weights` parameter. **model_type** - type of the model, 'Mobilenet3' and 'Mobilenet2' are available. **size** param means the size of the mobilenetv3, there are 'large' and 'small' options. Note that this will change mobilenev3 only. **embeding_dim** - the size of the embeding (vector of features after average pooling). **width_mult** - the width scaling parameter of the model. Note, that you will need the appropriate imagenet weights if you want to train your model with transfer learning. On google drive weights with 0.75, 1.0 value of this parameter is available
+* **aug** - advanced augmentation, appropriate value for type is 'cutmix' or 'mixup. lambda = BetaDistribution(alpha, beta), cutmix_prob - probability of applying cutmix on image.
```

Comment:
```suggestion
* **aug** - advanced augmentation, appropriate value for type is 'cutmix' or 'mixup'. lambda = BetaDistribution(alpha, beta), cutmix_prob - probability of applying cutmix on image.
```
Also, consider paraphrasing this sentence, as it may be hard to understand